manifests/d/DearMob&Digiarty,Inc/DearMobiPhoneManager/6.0.0.0/DearMob&Digiarty,Inc.DearMobiPhoneManager.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: DearMob&Digiarty,Inc.DearMobiPhoneManager
+PackageVersion: 6.0.0.0

```

```suggestion
PackageIdentifier: DearMob.iPhoneManager
PackageVersion: 6.0
```
Package version should match control panel